.github/actions/test-component/action.yml

```diff
@@ -1,5 +1,5 @@
-name: test specific sktime component
-description: run unit tests on individual component of sktime framework
+name: ${{ inputs.sktime-component-identifier }}, ${{ inputs.python-version-identifier }}
+description: ${{ inputs.sktime-component-identifier }}, ${{ inputs.python-version-identifier }}
```

Comment:
Same as above, does description have access to inputs?

Reply:
see above, reverted